src/screens/Home/Home.js

```diff
@@ -452,7 +456,7 @@ class HomeScreen extends React.Component<Props, State> {
             <React.Fragment>
               <ScrollView
                 style={{ width: '100%', flex: 1 }}
-                stickyHeaderIndices={referralsFeatureEnabled ? [4] : [3]}
+                stickyHeaderIndices={referralsFeatureEnabled ? [3] : [2]}
```

Comment:
What do these numbers mean exactly? Can we put a name on them?

Reply:
It means an index of items that should become sticky. This has been changed after ActivityFeed improvements